src/Cryptol/Parser/Unlit.hs

```diff
@@ -105,7 +105,7 @@ markdown = blanks []
 
   isOpenFence l
     | "```" `Text.isPrefixOf` l' =
-      Just $ case Text.drop 3 l' of
+      Just $ case Text.strip (Text.drop 3 l') of
```

Comment:
Would it suffice to just use `Text.dropWhile isSpace` here instead of `Text.strip`? The latter strips off whitespace from the end of the line, but I don't think that is necessary here, given that the code is already checking is `cryptol` is a prefix. As a result, it shouldn't matter if there is trailing whitespace at the end. (Please correct me if I've overlooked something, of course.)

Reply:
Nope, that's right :) I'll make that tweak and push...